cmd/flowserver/flow.go

```diff
@@ -44,10 +44,11 @@ func (r *flowResponse) MarshalJSON() ([]byte, error) {
 }
 
 type startRequest struct {
-	Flows   json.RawMessage      `json:"flows"    validate:"required"`
-	Contact json.RawMessage      `json:"contact"  validate:"required"`
-	Extra   json.RawMessage      `json:"extra,omitempty"`
-	Input   *utils.TypedEnvelope `json:"input"`
+	Environment *json.RawMessage     `json:"env"`
```

Comment:
For the json format, i'd like to stick with verbose. Those names can all compress down to nothing if we want at some point, so keeping them as easily human readable as possible sounds ideal.